### PR TITLE
Create manifest for wkhtmltox

### PR DESCRIPTION
**Pull request by bquistorff**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/28137)

### CODE REVIEW

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/28137#issuecomment-923323571

EDIT: Please sign the CLA.

**Comment by Trenly** [comment]
@ItzLevvie Author has signed the CLA now

File: manifests/w/wkhtmltopdf/wkhtmltox/0.12.6-1/wkhtmltopdf.wkhtmltox.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Created using YamlCreate.ps1 v1.1.7
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: wkhtmltopdf.wkhtmltox
+PackageVersion: 0.12.6-1
+MinimumOSVersion: 10.0.0.0
+FileExtensions:
+- html
+Commands:
+- wkhtmltopdf wkhtmltoimage
```
Comment (ItzLevvie) [inline manifests/w/wkhtmltopdf/wkhtmltox/0.12.6-1/wkhtmltopdf.wkhtmltox.installer.yaml:10]:
```suggestion
- wkhtmltopdf
- wkhtmltoimage
```

File: manifests/w/wkhtmltopdf/wkhtmltox/0.12.6-1/wkhtmltopdf.wkhtmltox.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+PackageUrl: https://wkhtmltopdf.org
+License: GPL
+LicenseUrl: https://github.com/wkhtmltopdf/wkhtmltopdf/blob/master/LICENSE
+#Copyright: 
+#CopyrightUrl: 
+ShortDescription: Convert HTML to PDF using Webkit (QtWebKit)
+#Description: 
+Moniker: wkhtmltox
+Tags:
+- html pdf
```
Comment (ItzLevvie) [inline manifests/w/wkhtmltopdf/wkhtmltox/0.12.6-1/wkhtmltopdf.wkhtmltox.locale.en-US.yaml:22]:
```suggestion
- html
- pdf
```

File: manifests/w/wkhtmltopdf/wkhtmltox/0.12.6-1/wkhtmltopdf.wkhtmltox.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: nullsoft
+#  Scope: 
+  InstallerUrl: https://github.com/wkhtmltopdf/packaging/releases/download/0.12.6-1/wkhtmltox-0.12.6-1.msvc2015-win64.exe
+  InstallerSha256: 1EFCFAF8472871268FD399865A3E62B435B91D4413009B80830EAAFF7A57D088
```
Comment (ItzLevvie) [inline manifests/w/wkhtmltopdf/wkhtmltox/0.12.6-1/wkhtmltopdf.wkhtmltox.installer.yaml:17]:
```suggestion
  InstallerSha256: 1EFCFAF8472871268FD399865A3E62B435B91D4413009B80830EAAFF7A57D088
  ProductCode: wkhtmltopdf
```

File: manifests/w/wkhtmltopdf/wkhtmltox/0.12.6-1/wkhtmltopdf.wkhtmltox.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+MinimumOSVersion: 10.0.0.0
+FileExtensions:
+- html
+Commands:
+- wkhtmltopdf wkhtmltoimage
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: nullsoft
+#  Scope:
```
Comment (ItzLevvie) [inline manifests/w/wkhtmltopdf/wkhtmltox/0.12.6-1/wkhtmltopdf.wkhtmltox.installer.yaml:15]:
```suggestion
  Scope: machine
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**